### PR TITLE
requestfinder: ConfigParser has been renamed to configparser in python3

### PR DESCRIPTION
**Pull request by nilxam**

### CODE REVIEW

**Comment by nilxam** [comment]
just a question: should we keep the code python2 compatible? or just go ahead to python3 only?

**Comment by coolo** [comment]
we're python3 only - the package buildrequires python3-osc

File: requestfinder.py [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
+#!/usr/bin/python3
 
-from ConfigParser import ConfigParser
+from configparser import ConfigParser
 from xdg.BaseDirectory import load_first_config
 from xml.etree import cElementTree as ET
+
 import sys
 import cmdln
 import logging
```
Comment (mimi1vx) [inline requestfinder.py]: can be remove too

File: requestfinder.py [hunk 2]
```diff
@@ -1,29 +1,25 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
-from ConfigParser import ConfigParser
+from configparser import ConfigParser
 from xdg.BaseDirectory import load_first_config
 from xml.etree import cElementTree as ET
```
Comment (mimi1vx) [inline requestfinder.py]: if is python3 only "his module will use a fast implementation whenever available. The xml.etree.cElementTree module is deprecated." from python 3.3

Comment (nilxam) [reply requestfinder.py]: It still exists there for backward compatible, that's why you can found cElementTree everywhere  in this repo, maybe at some point we should change it by another PR.

**Comment by coolo** [comment]
please rebase

**Comment by coolo** [comment]
that rebases? wow :)

**Comment by nilxam** [comment]
rebase what? it's a commit only...

**Comment by coolo** [comment]
I just wanted a green travis - I guess you're confusing it with squashing. Normally I git rebase origin/master and push -f, but your way worked too, as travis tests your commit against master

**Pull request merged**